packages/plugins/users-permissions/admin/src/pages/Roles/pages/ListPage/index.jsx

```diff
@@ -186,8 +185,7 @@ export const RolesListPage = () => {
                 })}
               </LinkButton>
             ) : null
-          }
-          navigationAction={<BackButton />}
+          }          
```

Comment:
Hey Sam, thank you for your contribution! I performed QA, and it's working.

Just one thing, could you run `yarn prettier:write` at the project root and `yarn lint` again before submitting your code? It seems like line 188 has some whitespaces, and it won't pass our linter. Once this is done we can merge it directly into develop. thank you! 😄

Reply:
@butcherZ , Thank you!  I’ve made the requested updates. Please let me know if there’s anything else is needed before merging!